### PR TITLE
docs(Configuration): fix document amdContainer

### PR DESCRIPTION
**Pull request by long76**
Fix `amdContainer` description

### CODE REVIEW

**Comment by chenxsan** [comment]
@TheLarkInn @alexander-akait Hey guys, it would be great if you could help review this pull request. I realized that I may have misunderstood something based on this comment: https://github.com/webpack/webpack.js.org/issues/6742#issuecomment-1500831469.

File: src/content/configuration/output.mdx [hunk 1]
```diff
@@ -900,7 +901,9 @@ In the above example, we're passing a single entry file to `entry`, however, web
 
 <Badge text="5.78.0+" />
 
-Define a namespace for `define`/`require` functions in the AMD module.
+Use a container(defined in global space) for call `define`/`require` functions in the AMD module.
```
Comment (TheLarkInn) [inline src/content/configuration/output.mdx:904]:
```suggestion
Use a container(defined in global space) for calling `define`/`require` functions in an AMD module.
```

File: src/content/configuration/output.mdx [hunk 2]
```diff
@@ -900,7 +901,9 @@ In the above example, we're passing a single entry file to `entry`, however, web
...
+
+W> Note that value of `amdContainer` **must be** set as global variable.
```
Comment (TheLarkInn) [inline src/content/configuration/output.mdx:906]:
```suggestion
W> Note that the value of `amdContainer` **must be** set as a global variable.
```

File: src/content/configuration/output.mdx [hunk 3]
```diff
@@ -914,14 +917,10 @@ module.exports = {
 };
 ```
 
-Which would result code like below:
+Which would result bundle like below:
```
Comment (TheLarkInn) [inline src/content/configuration/output.mdx:920]:
```suggestion
Which will result in the following bundle:
```

**Review by TheLarkInn — CHANGES_REQUESTED** [review]
Looks great! Just a few small wording changes and LGTM!

**Review by TheLarkInn — APPROVED** [review]

**Comment by TheLarkInn** [comment]
You can merge as you see fit @chenxsan. Thank you for the ping!

**Comment by long76** [comment]
Thanks!

**Pull request merged**